### DISCUSSION TITLE
Overview mode similar to revealjs ('Esc' key)

### DISCUSSION THREAD

**Discussion by arseru** [marp-team/marp#326] [Q&A] [answered]
Hi Marp team!

First of all, thanks so much for this wonderful tool, I'm using it to create my PhD defense presentation, and it works very well.

I was trying to find a way to have an overview mode of the slides (like it's done in [revealjs](https://revealjs.com/overview/)), to be able to move quickly between them, but I couldn't find anything in the documentation or in the discussions here.

Do you know if it is possible to display this mode in Marpit somehow?

**Comment by yhatt** [comment]
Tools provided by Marp team are not providing the solution like overview mode because we are focusing only to the experience of making slide. We assume to use an exported PDF/PPTX with [the other presentation tools](https://en.wikipedia.org/wiki/Comparison_of_PDF_presentation_software). (e.g. [pdfpc](https://pdfpc.github.io/) has an overview mode)

If you really need a overview mode in HTML, create an issue or pull request at [Marp CLI](https://github.com/marp-team/marp-cli) repository. It would become a one of features in [the `bespoke` template](https://github.com/marp-team/marp-cli#template).

**Reply by arseru** [reply]
Thanks for the answer yhatt! I've just opened an issue as per one of your suggestions :) https://github.com/marp-team/marp-cli/issues/464

**Reply by mil-ad** [reply]
I have a PR for this in marp-team/marp-cli#705

**Comment by yhatt** [comment, marked as answer]
The overview is available in the HTML with `bespoke` template, from [Marp CLI v4.3.0](https://github.com/marp-team/marp-cli/releases/tag/v4.3.0)!

Toggle it with the <kbd>o</kbd> key, <kbd>Esc</kbd> key, or overview button in the on-screen controls. In presenter view, click the text of the slide page to toggle as well.

![Image](https://github.com/user-attachments/assets/52a5a7ad-0fb7-4d1d-aa20-e5920146dddd)

In the overview mode, you can jump to the slide page by clicking the slide thumbnail. You can also use ULDR keys to navigate through the slides.
